emission/net/api/utility_model_api.py

```diff
@@ -0,0 +1,54 @@
+import emission.user_model_josh.utility_model as eum
+import emission.core.get_database as edb
+
+import json
+
+def write_to_db_from_browser(info):
```

Comment:
not sure this is the best name for the function. How about something that reflects what is it actually trying to do?